### DISCUSSION TITLE
Not able to see all the plays in local machine

### DISCUSSION THREAD

**Discussion by harshsinghatz** [reactplay/react-play#370] [Q&A]
![image](https://user-images.githubusercontent.com/51085015/178211683-d19ea257-0474-4214-9fa8-65c62af8da89.png)

**Comment by atapas** [comment]
Hey @harshsinghatz Good to see you here:

I think this step is missing.. Please let me know

https://github.com/reactplay/react-play/blob/main/README.md#%EF%B8%8F-create-a-env-file-at-the-root-of-your-project-folder-with-the-following-content

Btw, you can join our Discord too :)

https://discord.gg/vrTxWUP8Am

**Reply by james-david-marks** [reply]
I am having a similar experience. I discovered the .env file and that addresses the "Oops Play not Found" error. But it does not load my play and when i try to "force" execution of my play by going to the url http://localhost/plays/myplay it does not load. I debugged/inspected PlayList.jsx and found that as it iterates through plays?.map((play, index) it is finding things that are not in my local plays library like "CryptoCurrency..." and I tried commenting out all of the imports in the play-meta.js file but it is still loading. I have 29 plays but it is loading 31. So....all of this leads me to believe that it is loading the plays from some other location and not from my local library....NOTE: i can edit existing plays in my local library and it will reflect the changes so this seems to be limited to loading/registering plays. Thoughts?

**Reply by atapas** [reply]
> I am having a similar experience. I discovered the .env file and that addresses the "Oops Play not Found" error. But it does not load my play and when i try to "force" execution of my play by going to the url http://localhost/plays/myplay it does not load. I debugged/inspected PlayList.jsx and found that as it iterates through plays?.map((play, index) it is finding things that are not in my local plays library like "CryptoCurrency..." and I tried commenting out all of the imports in the play-meta.js file but it is still loading. I have 29 plays but it is loading 31. So....all of this leads me to believe that it is loading the plays from some other location and not from my local library....NOTE: i can edit existing plays in my local library and it will reflect the changes so this seems to be limited to loading/registering plays. Thoughts?

Hey @james-david-marks 

can you please create a PR with your changes? I can have a look.

**Reply by james-david-marks** [reply]
Yes. Let me try. I am pretty good with Git but not sure the best way to to a PR. I'm using command line. first i do a git add and then a git commit to my fork? then i go to github and create a pull request?

**Reply by james-david-marks** [reply]
#378 if you would take a look at it that would be great. the .env must have been on the ignore list but this is what I have inside:
REACT_APP_NHOST_BACKEND_URL=https://rgkjmwftqtbpayoyolwh.nhost.run/
REACT_APP_NHOST_VERSION=v1
REACT_APP_NHOST_ENDPOINT=graphql

**Reply by atapas** [reply]
Sure will do.

cc: @koustov

**Reply by james-david-marks** [reply]
I'm using npm  by the way. do you think it's different with yarn?

**Reply by atapas** [reply]
Npm is fine, yarn just a bit faster.

**Reply by james-david-marks** [reply]
Are there other backends? This is the one I am using: REACT_APP_NHOST_BACKEND_URL=https://rgkjmwftqtbpay

I don't think this is the right backend for local development. In the CreatePlay readme, it says that my play should appear in the playlist. But this is not building the playlist from my local src/plays folder. There are three plays that i do not have locally:

Cryptocurrency Tracker
Budget App
DevTestPlay15

When I click on on of these it says: Play is Under Development

**Comment by koustov** [comment]
Regarding the env file, there is a [documented step](https://github.com/reactplay/react-play#%EF%B8%8F-create-a-env-file-at-the-root-of-your-project-folder-with-the-following-content) to add the entries. Do you see any issue with that?

**Comment by koustov** [comment]
As per our current flow, `under-development` plays may appear on local and preview builds however not in production. We are doing some structural changes (under testing), and after that, this will be resolved

**Comment by harshsinghatz** [comment]
I can see all the plays after adding the .env but not able to see the play that i created

**Reply by atapas** [reply]
Hey @harshsinghatz 

As @koustov mentioned, it is due to the structural changed we are making in the create flow.

Could you please use this link to get your identity created?
https://rgkjmwftqtbpayoyolwh.nhost.run/v1/auth/signin/provider/github

After that, raise your PR, we will enable your play. This process is manual in the time being until the create flow change makes it automatic, ETA next 3 4 days.

**Reply by atapas** [reply]
@harshsinghatz the new create flow is in(in beta). Do you want to give it a try?

**Reply by harshsinghatz** [reply]
Yeah, I would give it a try

**Reply by harshsinghatz** [reply]
What are the steps for the new create flow?

**Reply by atapas** [reply]
> What are the steps for the new create flow?

https://blog.reactplay.io/reactplay-create-a-play-using-the-new-workflow
